datafusion/core/tests/dataframe_functions.rs

```diff
@@ -155,11 +155,11 @@ async fn test_fn_approx_median() -> Result<()> {
     let expr = approx_median(col("b"));
 
     let expected = vec![
-        "+----------------------+",
-        "| APPROXMEDIAN(test.b) |",
-        "+----------------------+",
-        "| 10                   |",
-        "+----------------------+",
+        "+-----------------------+",
```

Comment:
As part of this PR, I have added `convert_camel_to_upper_snake` util for display of Aggregate function. Their display name is automatically, calculate from struct name. Hence display name of some of the existing aggregators is changed.